hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/util/DistCpUtils.java

```diff
@@ -596,6 +596,8 @@ public static CopyMapper.ChecksumComparison checksumsAreEqual(
     } else if (sourceChecksum.equals(targetChecksum)) {
       return CopyMapper.ChecksumComparison.TRUE;
     }
+    LOG.info("Checksum not equal. Source checksum: {}, target checksum: {}",
+        sourceChecksum, targetChecksum);
```

Comment:
you aren't putting the paths for which checksum mismatch happened, if there are thousands of file being copied and bunch of them log this.
How would you figure out who checksum didn't match

Reply:
In our failed jobs, the mismatch source and target path was printed by https://github.com/apache/hadoop/blob/trunk/hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/util/DistCpUtils.java#L633.

Thanks for pointing that out. The paths have also been added to DistCpUtils.